vue/package.json

```diff
@@ -15,10 +15,10 @@
     "@capacitor/core": "3.2.5",
     "@capacitor/haptics": "1.1.2",
```

Comment:
I don't think this is related to the PR, but I get a peer dependency issue on Node 16/npm 8.19.3. Do you get the same thing?

Reply:
I don't 🤔 Tried deleting `node_modules` and re-installing (specifically in the Vue app) and everything worked okay. Using Node 16.13.0 and npm 8.1.0. I could try updating my npm if you're still seeing it after a clean reinstall?

Reply:
I'm getting this:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: ts-jest@27.0.7
npm ERR! Found: @types/jest@24.9.1
npm ERR! node_modules/@types/jest
npm ERR!   dev @types/jest@"^24.0.19" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peerOptional @types/jest@"^27.0.0" from ts-jest@27.0.7
npm ERR! node_modules/ts-jest
npm ERR!   dev ts-jest@"^27.0.4" from the root project
npm ERR!   peerOptional ts-jest@"^27.0.4" from @vue/cli-plugin-unit-jest@5.0.0-beta.7
npm ERR!   node_modules/@vue/cli-plugin-unit-jest
npm ERR!     dev @vue/cli-plugin-unit-jest@"~5.0.0-beta.7" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: @types/jest@27.5.2
npm ERR! node_modules/@types/jest
npm ERR!   peerOptional @types/jest@"^27.0.0" from ts-jest@27.0.7
npm ERR!   node_modules/ts-jest
npm ERR!     dev ts-jest@"^27.0.4" from the root project
npm ERR!     peerOptional ts-jest@"^27.0.4" from @vue/cli-plugin-unit-jest@5.0.0-beta.7
npm ERR!     node_modules/@vue/cli-plugin-unit-jest
npm ERR!       dev @vue/cli-plugin-unit-jest@"~5.0.0-beta.7" from the root project
npm ERR! 
```

It looks like these are still using Vue CLI 5 beta dependencies which could explain it.

Reply:
Updated to the same version of npm as you and I'm now getting the same error. Looking into it 👀

Reply:
Looks like `npm install --legacy-peer-deps` works okay, if we're alright with that resolution?

Reply:
I think that's fine for the purposes of getting this PR merged, but we'll want to fix this in a future PR since the peer dependencies are incorrect.

Reply:
Cool, I'll make a ticket to address it 👍